app/services/billable_metrics/aggregations/recurring_count_service.rb

```diff
@@ -11,6 +11,19 @@ def aggregate(from_date:, to_date:, options: {})
         result
       end
 
+      def breakdown(from_date:, to_date:)
+        @from_date = from_date.to_date
+        @to_date = to_date.to_date
+
+        breakdown = persisted_breakdown
+        breakdown += added_breakdown
+        breakdown += removed_breadown
+        breakdown += added_and_removed_breakdown
+
+        result.breakdown = breakdown.sort_by(&:date)
```

Comment:
Probably a bit overkill, but we can get better complexity here (instead of using built-in sort) with custom sorting algorithm since each array is already sorted

Reply:
Nice one :thinking: 
I think we can keep it as it for now since we will probably have really few persisted events per customer. We can keep it in mind for future performance tweak if required. What do you think?

Reply:
Yep make sense. We can improve sorting part from `O(n*logn) -> O(n)` if `n` is the length of the final array. But yeah, it wouldn't be significant if there is small number of elements